File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/DetallePedido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class DetallePedido
    {
        [Key]
        public int idDetalleOrden { get; set; }

        public int cantidad { get; set; }

        public int idPedido { get; set; }
        [ForeignKey("idPedido")]
        public Pedido Ordenes { get; set; }

        public int idMenu { get; set; }
        [ForeignKey("idMenu")]
        public Menu Menu { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/DetallePedido.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class DetallePedido
    <Key>
    Public Property idDetalleOrden As Integer
    Public Property cantidad As Integer
    Public Property idPedido As Integer
    <ForeignKey("idPedido")>
    Public Property Ordenes As Pedido
    Public Property idMenu As Integer
    <ForeignKey("idMenu")>
    Public Property Menu As Menu
End Class
